bedrock/newsletter/templates/newsletter/index.html

```diff
@@ -12,7 +12,11 @@
     {# Remove the span tags from the headline next time it changes: #}
     <h1 class="large">{{ _('Read all about it in our <span>newsletter</span>') }}</h1>
     <p>
-      {% trans %}Subscribe to monthly updates and keep current with Mozilla news, including the latest tips and tricks for getting the most out of your Firefox browser. It’s the perfect way for us to keep in touch!{% endtrans %}
+      {% if l10n_has_tag('mozilla-newsletter-2017') %}
```

Comment:
nit: Can you use underscores for the tag? `mozilla_newsletter_2017` would be fine (we should add that to the doc). The rest of the code is correct.